### PR TITLE
[WIP]Bug 1318737, added Recovering Deleted Default Projects section

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1318737

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@liggitt @adellape @tnguyen-rh PTAL. Thanks!

File: admin_guide/managing_projects.adoc [hunk 1]
```diff
@@ -309,3 +309,26 @@ in order, this rule should be specified last.
...
+
+====
+----
+# systemctl restart atomic-openshift-master
+----
+====
+
+The master creates the missing namespaces and bindings on startup.
+
+Then, you need to manually recreate the content (including templates, image
```
Comment (tnguyen-rh) [inline admin_guide/managing_projects.adoc]: @ahardin-rh 
Suggest s/Then/Next/

**Comment by tnguyen-rh** [comment]
@ahardin-rh Small suggestion, otherwise LGTM.

**Comment by ahardin-rh** [comment]
@liggitt PTAL. Thanks!

File: admin_guide/managing_projects.adoc [hunk 2]
```diff
@@ -309,3 +309,26 @@ in order, this rule should be specified last.
...
+
+== Recovering Deleted Default Projects
+
+If you delete any of the default projects like `*default*`, `*openshift*`, or
+`*openshift-infra*`, then you will likely experience issues that impact the
+whole environment. To recover from this, restart the master service:
+
+====
+----
+# systemctl restart atomic-openshift-master
```
Comment (liggitt) [inline admin_guide/managing_projects.adoc]: after restarting, verify the projects exist and are not in the terminating state:
```
oc get namespace/default namespace/openshift namespace/openshift-infra
```

If they are still terminating, wait for the termination to complete, then restart the master service again.

**Comment by ahardin-rh** [comment]
@liggitt Thanks! Made some edits.

**Comment by ahardin-rh** [comment]
@liggitt PTAL at QE's comments in the BZ for this work: 
https://bugzilla.redhat.com/show_bug.cgi?id=1318737
I'm not sure how to address this. Any thoughts?

**Comment by liggitt** [comment]
looking at this again, recovering the deleted openshift-infra project will be very difficult... restarting won't help in 3.2. https://github.com/openshift/origin/pull/4318 is opened to protect those two namespaces from deletion.

**Comment by ahardin-rh** [comment]
@liggitt Thanks. I made an edit regarding openshift-infra. However, to be clear, do you think that the recovery steps are not worth documenting here? Do you think that those would be better suited for a KB article?

**Comment by ahardin-rh** [comment]
@liggitt bump

**Comment by liggitt** [comment]
I probably wouldn't put it in the core docs here. With the fix in https://github.com/openshift/origin/pull/4318 it won't be possible to completely delete those namespaces any more

**Comment by ahardin-rh** [comment]
@liggitt  Thanks. Do you think we should add this for 3.0 and 3.1 branches only (omitting this from Origin and 3.2)?

**Comment by liggitt** [comment]
I wouldn't put it in core docs

**Comment by ahardin-rh** [comment]
@liggitt Thanks. Moved it to KB: https://access.redhat.com/solutions/2260211

**Pull request closed without merging**